clustered/server/entity/src/test/java/org/ehcache/clustered/server/store/ClusterTierActiveEntityTest.java

```diff
@@ -880,7 +884,7 @@ public void testDataSyncToPassiveCustomBatchSize() throws Exception {
         @SuppressWarnings("unchecked")
         PassiveSynchronizationChannel<EhcacheEntityMessage> syncChannel = mock(PassiveSynchronizationChannel.class);
         activeEntity.synchronizeKeyToPassive(syncChannel, concurrencyKey);
-        verify(syncChannel, atLeast(2)).synchronizeToPassive(any(EhcacheDataSyncMessage.class));
+        verify(syncChannel, atLeast(1)).synchronizeToPassive(any(EhcacheDataSyncMessage.class));
```

Comment:
This only needs changing because the code has invoke context issues. This might be fixed in my PR I haven't checked.

Reply:
these were because there used to be a tracker sync with every segment.  Now the tracker sync is only sent at the very beginning.